privaterelay/templates/includes/premium-onboarding.html

```diff
@@ -0,0 +1,127 @@
+{% load static %}
+{% load ftl %}
+{% ftlconf bundle='privaterelay.ftl_bundles.main' %}
+
+<!-- <button class="c-premium-onboarding-dismiss js-premium-onboarding-dismiss"><img src="{% static 'images/x-close.svg' %}" alt="Close"></button> -->
+
+<!-- TODO: Remove/port classes from .c-premium-onboarding  -->
+<div class="c-premium-onboarding c-multipart-premium-onboarding">
+    <ul class="c-premium-onboarding-steps">
```

Comment:
Nit, but I think this should be an ordered lists, since users should be going through the steps in order :) Also for the `<ul>` below.

Reply:
Revised in e4ab4ee8f06e1a61deaf6de33cafd6be95d0e3de